include/dronecore.h

```diff
@@ -77,6 +78,40 @@ class DroneCore
     ConnectionResult add_udp_connection(int local_port_number);
 
     /**
+     * @brief Adds a TCP connection with the default arguments.
+     *
+     * This will connect to local TCP port 5760.
+     *
+     * @return The result of adding the connection.
+     */
+    ConnectionResult add_tcp_connection();
```

Comment:
Just For my information, why don't we have single function with default arguments for these connection APIs?